scalableminds/webknossos
# Fix click/trace offset in orthogonal mode

### URL of deployed dev instance (used for testing):
- https://fixclickoffset.webknossos.xyz

### Steps to test:
- Open Hybrid tracing. In Trace mode the contour should follow closely to the mouse pointer, there should be no offset.
- In skeleton mode, activate Soma-Clicking and set nodes using `Ctrl + Click` (especially near the borders). The nodes should be created where you click with as little as possible offset. (It's not perfect yet but better)

------
